Sweetener.Logging/LogLevel.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace Sweetener.Logging
+{
+    /// <summary>
+    /// Specifies a priority level and semantics associated with a log entry.
+    /// </summary>
+    public enum LogLevel
```

Comment:
Does anyone feel strongly about changing the names of the levels OR the names of the logging methods?